project/app/Models/Role.php

```diff
@@ -8,4 +8,6 @@
 class Role extends Model
 {
     use HasFactory;
+    public $timestamps = false;
```

Comment:
Again: dlaczego ta linijka jest potrzebna?

Reply:
Coś tam się działo, że przy aktualizacji, chciał wpisywać timestamp z jej datą, a stwierdziłem, że to niepotrzebne i usunałem to.